### PR TITLE
Renaming to Trigger Code Reference

### LINKED ISSUE #1802: Update references to Trigger Code Reference (TCR) instead of vocab-mapper

### PR DESCRIPTION
**Pull request by robertandremitchell**
# PULL REQUEST

## Summary
Removing and updating references from `vocab-mapper` to `trigger-reference-code`
## Related Issue
Fixes #1802 

## Additional Information
Idk, the endpoint is a little wordy imo. Do we want to shorten it? And if so, what would be appropriate?

## Checklist

- [ ] If this code affects the other scrum team, have they been notified? (In Slack, as reviewers, etc.)

[//]: # (PR title: Remember to name your PR descriptively!)

### CODE REVIEW

File: containers/trigger-code-reference/description.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+## bGetting Started with the DIBBs Trigger Code Reference  Service
+
+### Introduction
+
+The DIBBs  Trigger Code Reference (trigger-code-reference) service offers a REST API devoted to querying and enriching SNOMED condition code analysis. This service stores condition codes and their associated value sets so that they can be queried by users, as well as inserted into supplied FHIR bundles as tagged extensions for future path parsing.
```
Comment (DanPaseltiner) [inline containers/trigger-code-reference/description.md:5]: Is there an extra space about "The DIBBs"?

**Review by DanPaseltiner — APPROVED** [review]
LGTM 🚀

File: containers/trigger-code-reference/description.md (hunk 1 above)
Comment (bamader) [reply containers/trigger-code-reference/description.md:5]: Yep, I see it too.

**Review by bamader — APPROVED** [review]
LGTM!

File: containers/trigger-code-reference/description.md (hunk 1 above)
Comment (robertandremitchell) [reply containers/trigger-code-reference/description.md]: updated!

**Pull request merged**